src/Microsoft.OpenApi/Validations/Rules/OpenApiExternalDocsRules.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Properties;
+
+namespace Microsoft.OpenApi.Validations.Rules
+{
+    /// <summary>
+    /// The validation rules for <see cref="OpenApiExternalDocs"/>.
+    /// </summary>
+    [OpenApiRule]
+    internal static class OpenApiExternalDocsRules
+    {
+        /// <summary>
+        /// Validate the field is required.
+        /// </summary>
+        public static ValidationRule<OpenApiExternalDocs> FieldIsRequired =>
+            new ValidationRule<OpenApiExternalDocs>(
+                (context, item) =>
+                {
+                    // title
```

Comment:
>title [](start = 23, length = 5)

url

Reply:
Thx.

---
In reply to: [157651258](https://github.com/Microsoft/OpenAPI.NET/pull/152#discussion_r157651258) [](ancestors = 157651258)